src/ActiveRoomObserver.js

```diff
@@ -51,23 +55,23 @@ class ActiveRoomObserver {
         }
     }
 
-    _emit(roomId) {
```

Comment:
fun that ts/flow annotation works even in a non-flow js file

Reply:
it's both great and awful.